packages/number-field/src/NumberField.ts

```diff
@@ -759,5 +770,15 @@ export class NumberField extends TextfieldBase {
             }
             this.inputElement.inputMode = inputMode;
         }
+        if (
+            changes.has('focused') &&
+            this.focused &&
+            !this.hasRecentlyReceivedPointerDown &&
+            !!this.formatOptions.unit
+        ) {
+            // Normalize keyboard focus entry between unit and non-unit bearing Number Fields
+            this.setSelectionRange(0, this.displayValue.length);
+            this.hasRecentlyReceivedPointerDown = false;
```

Comment:
Is it necessary to again assign `this.hasRecentlyReceivedPointerDown = false;`, when it was already false when it was checked on line 776?

Reply:
Great question. I feel like that was required at some point, but it _shouldn't_ do anything in this version. Will confirm.

Reply:
Removed.